src/Symfony/Component/Validator/composer.json

```diff
@@ -62,6 +63,7 @@
         "symfony/yaml": "",
         "symfony/config": "",
         "egulias/email-validator": "Strict (RFC compliant) email validation",
+        "bjeavons/zxcvbn-php": "Password strength estimation used by the PasswordStrength constraint",
```

Comment:
Let's remove this one. It's not needed, and discovery happens with docs and/or auto-completion, ... but never via the suggest Composer thingy IMHO.

Reply:
OK I understand. I removed this line.